app/controllers/api/public/v1/courses_controller.rb

```diff
@@ -19,7 +19,9 @@ def courses
         end
 
         def recruitment_cycle
-          @recruitment_cycle ||= RecruitmentCycle.find_by!(year: params[:recruitment_cycle_year])
+          @recruitment_cycle = RecruitmentCycle.find_by(
```

Comment:
```suggestion
          @recruitment_cycle ||= RecruitmentCycle.find_by(
```
can we put the memoization back in? just it incase it gets called again it doesn't have to make another API request

Reply:
yup! copy and paste fail 🤦‍♂️